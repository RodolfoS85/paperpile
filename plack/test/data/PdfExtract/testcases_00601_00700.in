---
file: ../../../PDFREPOSITORY/000601.pdf
---
file: ../../../PDFREPOSITORY/000602.pdf
---
file: ../../../PDFREPOSITORY/000603.pdf
---
file: ../../../PDFREPOSITORY/000604.pdf
---
file: ../../../PDFREPOSITORY/000605.pdf
---
file: ../../../PDFREPOSITORY/000606.pdf
---
file: ../../../PDFREPOSITORY/000607.pdf
---
file: ../../../PDFREPOSITORY/000608.pdf
---
file: ../../../PDFREPOSITORY/000609.pdf
---
file: ../../../PDFREPOSITORY/000610.pdf
---
file: ../../../PDFREPOSITORY/000611.pdf
---
file: ../../../PDFREPOSITORY/000612.pdf
---
file: ../../../PDFREPOSITORY/000613.pdf
---
file: ../../../PDFREPOSITORY/000614.pdf
---
file: ../../../PDFREPOSITORY/000615.pdf
---
file: ../../../PDFREPOSITORY/000616.pdf
---
file: ../../../PDFREPOSITORY/000617.pdf
---
file: ../../../PDFREPOSITORY/000618.pdf
---
file: ../../../PDFREPOSITORY/000619.pdf
---
file: ../../../PDFREPOSITORY/000620.pdf
---
file: ../../../PDFREPOSITORY/000621.pdf
---
file: ../../../PDFREPOSITORY/000622.pdf
---
file: ../../../PDFREPOSITORY/000623.pdf
---
file: ../../../PDFREPOSITORY/000624.pdf
---
file: ../../../PDFREPOSITORY/000625.pdf
---
file: ../../../PDFREPOSITORY/000626.pdf
---
file: ../../../PDFREPOSITORY/000627.pdf
---
file: ../../../PDFREPOSITORY/000628.pdf
---
file: ../../../PDFREPOSITORY/000629.pdf
---
file: ../../../PDFREPOSITORY/000630.pdf
---
file: ../../../PDFREPOSITORY/000631.pdf
---
file: ../../../PDFREPOSITORY/000632.pdf
---
file: ../../../PDFREPOSITORY/000633.pdf
---
file: ../../../PDFREPOSITORY/000634.pdf
---
file: ../../../PDFREPOSITORY/000635.pdf
---
file: ../../../PDFREPOSITORY/000636.pdf
---
file: ../../../PDFREPOSITORY/000637.pdf
---
file: ../../../PDFREPOSITORY/000638.pdf
---
file: ../../../PDFREPOSITORY/000639.pdf
---
file: ../../../PDFREPOSITORY/000640.pdf
---
file: ../../../PDFREPOSITORY/000641.pdf
---
file: ../../../PDFREPOSITORY/000642.pdf
---
file: ../../../PDFREPOSITORY/000643.pdf
---
file: ../../../PDFREPOSITORY/000644.pdf
---
file: ../../../PDFREPOSITORY/000645.pdf
---
file: ../../../PDFREPOSITORY/000646.pdf
---
file: ../../../PDFREPOSITORY/000647.pdf
---
file: ../../../PDFREPOSITORY/000648.pdf
---
file: ../../../PDFREPOSITORY/000649.pdf
---
file: ../../../PDFREPOSITORY/000650.pdf
---
file: ../../../PDFREPOSITORY/000651.pdf
---
file: ../../../PDFREPOSITORY/000652.pdf
---
file: ../../../PDFREPOSITORY/000653.pdf
---
file: ../../../PDFREPOSITORY/000654.pdf
---
file: ../../../PDFREPOSITORY/000655.pdf
---
file: ../../../PDFREPOSITORY/000656.pdf
---
file: ../../../PDFREPOSITORY/000657.pdf
---
file: ../../../PDFREPOSITORY/000658.pdf
---
file: ../../../PDFREPOSITORY/000659.pdf
---
file: ../../../PDFREPOSITORY/000660.pdf
---
file: ../../../PDFREPOSITORY/000661.pdf
---
file: ../../../PDFREPOSITORY/000662.pdf
---
file: ../../../PDFREPOSITORY/000663.pdf
---
file: ../../../PDFREPOSITORY/000664.pdf
---
file: ../../../PDFREPOSITORY/000665.pdf
---
file: ../../../PDFREPOSITORY/000666.pdf
---
file: ../../../PDFREPOSITORY/000667.pdf
---
file: ../../../PDFREPOSITORY/000668.pdf
---
file: ../../../PDFREPOSITORY/000669.pdf
---
file: ../../../PDFREPOSITORY/000670.pdf
---
file: ../../../PDFREPOSITORY/000671.pdf
---
file: ../../../PDFREPOSITORY/000672.pdf
---
file: ../../../PDFREPOSITORY/000673.pdf
---
file: ../../../PDFREPOSITORY/000674.pdf
---
file: ../../../PDFREPOSITORY/000675.pdf
---
file: ../../../PDFREPOSITORY/000676.pdf
---
file: ../../../PDFREPOSITORY/000677.pdf
---
file: ../../../PDFREPOSITORY/000678.pdf
---
file: ../../../PDFREPOSITORY/000679.pdf
---
file: ../../../PDFREPOSITORY/000680.pdf
---
file: ../../../PDFREPOSITORY/000681.pdf
---
file: ../../../PDFREPOSITORY/000682.pdf
---
file: ../../../PDFREPOSITORY/000683.pdf
---
file: ../../../PDFREPOSITORY/000684.pdf
---
file: ../../../PDFREPOSITORY/000685.pdf
---
file: ../../../PDFREPOSITORY/000686.pdf
---
file: ../../../PDFREPOSITORY/000687.pdf
---
file: ../../../PDFREPOSITORY/000688.pdf
---
file: ../../../PDFREPOSITORY/000689.pdf
---
file: ../../../PDFREPOSITORY/000690.pdf
---
file: ../../../PDFREPOSITORY/000691.pdf
---
file: ../../../PDFREPOSITORY/000692.pdf
---
file: ../../../PDFREPOSITORY/000693.pdf
---
file: ../../../PDFREPOSITORY/000694.pdf
---
file: ../../../PDFREPOSITORY/000695.pdf
---
file: ../../../PDFREPOSITORY/000696.pdf
---
file: ../../../PDFREPOSITORY/000697.pdf
---
file: ../../../PDFREPOSITORY/000698.pdf
---
file: ../../../PDFREPOSITORY/000699.pdf
---
file: ../../../PDFREPOSITORY/000700.pdf
