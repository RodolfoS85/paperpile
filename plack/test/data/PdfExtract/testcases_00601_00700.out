---
authors: 'Jansen, Ronald and Yu, Haiyuan and Greenbaum, Dov and Kluger, Yuval and Krogan, Nevan J. and Chung, Sambath and Emili, Andrew and Snyder, Michael and Greenblatt, Jack F. and Gerstein, Mark'
title: A Bayesian Networks Approach for Predicting Protein-Protein Interactions from Genomic Data
---
authors: 'Jansson, Jesper and Hieu, Ngo Trung and Sung, Wing-Kin'
title: Local Gapped Subforest Alignment and Its Application in Finding RNA Structural Motifs
---
authors: 'Jenkins, Morgan and Keir, Mary and McCune, Joseph M.'
title: A Membrane-bound Fas Decoy Receptor Expressed by Human Thymocytes
---
authors: 'Jensen, Lars Juhl and Knudsen, Steen'
title: Automatic discovery of regulatory patterns in promoter regions based on whole cell expression data and functional annotation
---
authors: 'Ji, Hongkai and Zhou, Qing and Wen, Fang and Xia, Huiyu and Lu, Xin and Li, Yanda'
title: 'AsMamDB: an alternative splice database of mammals'
---
authors: 'Jiang, Zhihua and Arner, Elias S. J. and Mu, Ying and Johansson, Linda and Shi, Jinming and Zhao, Siqi and Liu, Shujun and Wang, Ruiying and Zhang, Tianzhu and Yan, Ganglin and Liu, Junqiu and Shen, Jiacong and Luo, Guimin'
doi: 10.1016/j.bbrc.2004.06.110
title: Expression of selenocysteine-containing glutathione S-transferase in Escherichia coli
---
authors: 'Jiang, Tao and Lin, Guohui and Ma, Bin and Zhang, Kaizhong'
title: A General Edit Distance between RNA Structures
---
authors: 'Lin, Guohui and Chen, Zhi-Zhong and Jiang, Tao and Wen, Jianjun'
title: The longest common subsequence problem for sequences with nested arc annotations
---
authors: 'Jirapech-Umpai, Thanyaluk and Aitken, Stuart'
doi: 10.1186/1471-2105-6-148
title: 'Feature selection and classification for microarray data analysis: Evolutionary methods for identifying predictive genes'
---
authors: 'Jobes, David V. and Thien, Leonard B.'
title: A Conserved Motif in the 5.8S Ribosomal RNA (rRNA) Gene is a Useful Diagnostic Marker for Plant Internal Transcribed Spacer (ITS) Sequences
---
authors: 'John, Bino and Enright, Anton J. and Aravin, Alexei and Tuschl, Thomas and Sander, Chris and Marks, Debora S.'
doi: 10.1371/journal.pbio.0020363
title: Human MicroRNA Targets
---
authors: 'Johnson, Jason M. and Castle, John and Garrett-Engele, Philip and Kan, Zhengyan and Loerch, Patrick M. and Armour, Christopher D. and Santos, Ralph and Schadt, Eric E. and Stoughton, Roland and Shoemaker, Daniel D.'
title: Genome-Wide Survey of Human Alternative Pre-mRNA Splicing with Exon Junction Microarrays
---
authors: 'Jonassen, Inge and Collins, John E and Higgins, Desmond G.'
title: Finding flexible patterns in unaligned protein sequences
---
authors: 'Jones, David T'
title: Protein structure prediction in the postgenomic era
---
authors: 'Jones, David T.'
title: Do transmembrane protein superfolds exist?
---
authors: 'Jones, Richard B. and Wang, Fen and Luo, Yongde and Yu, Chundong and Jin, Chengliu and Suzuki, Tohru and Kan, Mikio and McKeehan, Wallace L.'
doi: 10.1074/jbc.M006151200
title: The Nonsense-mediated Decay Pathway and Mutually Exclusive Expression of Alternatively Spliced FGFR2IIIb and -IIIc mRNAs
---
authors: 'Jonsson, Henrik and Heisler, Marcus G. and Shapiro, Bruce E. and Meyerowitz, Elliot M. and Mjolsness, Eric'
doi: 10.1073/pnas.0509839103
title: An auxin-driven polarized transport model for phyllotaxis
---
authors: 'King Jordan, I. and Rogozin, Igor B. and Glazko, Galina V. and Koonin, Eugene V.'
title: Origin of a substantial fraction of human regulatory sequences from transposable elements
---
authors: 'Joun, Hoseung and Lanske, Beate and Karperien, Marcel and Qian, Fang and Defize, Libert and ABOU-SAMRA, Abdul'
title: 'Tissue-Specific Transcription Start Sites and Alternative Splicing of the Parathyroid Hormone (PTH)/PTH-Related Peptide (PTHrP) Receptor Gene: A New PTH/PTHrP Receptor Splice Variant that Lacks the Signal Peptide'
---
authors: 'Junker, V. L. and Apweiler, R. and Bairoch, A.'
title: Representation of functional information in the SWISS-PROT Data Bank
---
authors: 'Kabat, Jennifer L. and Barberan-Soler, Sergio and McKenna, Paul and Clawson, Hiram and Farrer, Tracy and Zahler, Alan M.'
doi: 10.1371/journal.pcbi.0020086
title: Intronic Alternative Splicing Regulators Identified by Comparative Genomics in Nematodes
---
authors: 'Kaberdin, Vladimir R. and Blasi, Udo'
doi: 10.1111/j.1574-6976.2006.00043.x
title: Translation initiation and the fate of bacterial mRNAs
---
authors: 'Kafri, Yariv and Mukamel, David and Peliti, Luca'
title: Why is the DNA Denaturation Transition First Order?
---
authors: 'Kall, Lukas and Krogh, Anders and Sonnhammer, Erik L. L.'
doi: 10.1016/j.jmb.2004.03.016
title: A Combined Transmembrane Topology and Signal Peptide Prediction Method
---
authors: 'Kall, Lukas and Sonnhammer, Erik L. L.'
title: Reliability of transmembrane predictions in whole-genome data
---
authors: 'Kalnina, Zane and Zayakin, Pawel and Silina, Karina and Line, Aija'
doi: 10.1002/gcc.20156
title: Alterations of Pre-mRNA Splicing in Cancer
---
authors: 'Kalyna, M. and Barta, A.'
title: 'A plethora of plant serine/arginine-rich proteins: redundancy or evolution of novel gene functions?'
---
authors: 'Moreno, Sergio and Sohrmann, Marc and Welchman, David P. and Zipperlen, Peder and Ahringer, Julie'
title: Systematic functional analysis of the Caenorhabditis elegans genome using RNAi
---
authors: 'Kamatkar, Shubhangi and Radha, Vegesna and Nambirajan, S. and Reddy, R. Sreekantha and Swarup, Ghanshyam'
title: 'Two Splice Variants of a Tyrosine Phosphatase Differ in Substrate Specificity, DNA Binding, and Subcellular Location'
---
authors: 'Kan, Zhengyan and Gish, Warren and Rouchka, Eric and Glasscock, Jarret and States, David'
title: UTR Reconstruction and Analysis Using Genomically Aligned EST Sequences
---
authors: 'Kan, Zhengyan and States, David and Gish, Warren'
doi: 10.1101/gr.764102
title: Selecting for Functional Alternative Splices in ESTs
---
authors: 'Kaplan, Tommy and Friedman, Nir and Margalit, Hanah'
doi: 10.1371/journal.pcbi.0010001
title: Ab Initio Prediction of Transcription Factor Targets Using Structural Knowledge
---
authors: 'Karas, Holger and Knuppel, Rainer and Schuiz, Wolfgang and Sklenar, Heinz and Wingender, Edgar'
title: Combining structural analysis of DNA with search routines for the detection of transcription regulatory elements
---
authors: 'Karchin, Rachel and Karplus, Kevin and Haussler, David'
title: Classifying G-protein coupled receptors with support vector machines
---
authors: 'Karinch, Anne M. and Demello, Daphne E. and Floros, Joanna'
title: Effect of genotype on the levels of surfactant protein A mRNA and on the SP-A2 splice variants in adult humans
---
authors: 'Karplus, Martin and Sali, Andrej'
title: Theoretical studies of protein folding and unfolding
---
authors: 'Karpova, Alla Y. and Howley, Peter M. and Ronco, Lucienne V.'
doi: 10.1101/gad.813800
title: Dual utilization of an acceptor/donor splice site governs the alternative splicing of the IRF-3 gene
---
authors: 'Kasrayan, Alex and Persson, Annika L. and Sahlin, Margareta and Sjoberg, Britt-Marie'
doi: 10.1074/jbc.M106538200
title: The Conserved Active Site Asparagine in Class I Ribonucleotide Reductase Is Essential for Catalysis
---
authors: 'Kataoka, Naoyuki and Yong, Jeongsik and Narry Kim, V. and Velazquez, Francisco and Perkinson, Robert A. and Wang, Fan and Dreyfuss, Gideon'
title: Pre-mRNA Splicing Imprints mRNA in the Nucleus with a Novel RNA-Binding Protein that Persists in the Cytoplasm
---
authors: 'Kato, Mamoru and Hata, Naoya and Banerjee, Nilanjana and Futcher, Bruce and Zhang, Michael Q'
title: Identifying combinatorial regulation of transcription factors and binding motifs
---
authors: 'Kato, Mikio and McAllister, Chad J. and Hokabe, Shingo and Shimizu, Nobuyoshi and Lyubchenko, Yuri L.'
doi: 10.1046/j.1432-1033.2002.03063.x
title: Structural heterogeneity of pyrimidine/purine-biased DNA sequence analyzed by atomic force microscopy
---
authors: 'Katoh, Kazutaka and Misawa, Kazuharu and Kuma, Kei-ichi and Miyata, Takashi'
title: 'MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform'
---
authors: 'Katoh, Kazutaka and Toh, Hiroyuki'
doi: 10.1186/1471-2105-9-212
title: Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework
---
authors: 'Katti, Mukund V. and Sakharkar, Meena K. and Ranjekar, Prabhakar K. and Gupta, Vidya S.'
title: 'TRES: comparative promoter sequence analysis'
---
authors: 'Katz, Luba and Burge, Christopher B.'
doi: 10.1101/gr.1257503
title: Widespread Selection for Local RNA Secondary Structure in Coding Regions of Bacterial Genes
---
authors: 'Kawamoto, Hiroshi and Koide, Yukari and Morita, Teppei and Aiba, Hiroji'
doi: 10.1111/j.1365-2958.2006.05288.x
title: Base-pairing requirement for RNA silencing by a bacterial small RNA and acceleration of duplex formation by Hfq
---
authors: 'Kaya, Huseyin and Chan, Hue Sun'
title: Simple Two-State Protein Folding Kinetics Requires Near-Levinthal Thermodynamic Cooperativity
---
authors: 'Kececioglu, John and Starrett, Dean'
title: Aligning Alignments Exactly
---
authors: 'Kececioglu, John D. and Lenhof, Hans-Peter and Mehlhorn, Kurt and Mutzel, Petra and Reinert, Knut and Vingron, Martin'
title: A polyhedral approach to sequence alignment problems
---
authors: 'Keefe, Anthony D. and Szostak, Jack W.'
title: Functional proteins from a random-sequence library
---
authors: 'Keegan, Liam P. and Gallo, Angela and OConnell, Mary A.'
title: THE MANY ROLES OF AN RNA EDITOR
---
authors: 'Keich, U. and Pevzner, P. A.'
title: Finding motifs in the twilight zone
---
authors: 'Keil, Matthias and Exner, Thomas E. and Brickmann, Jurgen'
title: 'Pattern Recognition Strategies for Molecular Surfaces: III. Binding Site Prediction with a Neural Network'
---
authors: 'Kel-Margoulis, Olga V. and Romashchenko, Aida G. and Kolchanov, Nikolay A. and Wingender, Edgar and Kel, Alexander E.'
title: 'COMPEL: a database on composite regulatory elements providing combinatorial transcriptional regulation'
---
authors: 'Kel, Alexander and Kel-Margoulis, Olga and Babenko, Vladimir and Wingender, Edgar'
title: Recognition of NFATp/AP-1 Composite Elements within Genes Induced upon the Activation of Immune Cells
---
authors: 'Kel, Alexander E. and Kel-Margoulis, Olga V. and Farnham, Peggy J. and Bartley, Stephanie M. and Wingender, Edgar and Zhang, Michael Q.'
doi: 10.1006/jmbi.2001.4650
title: 'Computer-assisted Identification of Cell Cycle-related Genes: New Targets for E2F Transcription Factors'
---
authors: 'Kel, Alexdander and Voss, Nico and Jauregui, Ruy and Kel-Margoulis, Olga and Wingender, Edgar'
doi: 10.1186/1471-2105-7-S2-S13
title: 'Beyond microarrays: Finding key transcription factors controlling signal transduction pathways'
---
authors: 'Keresztes, Margit and Boonstra, Johannes'
title: Import(ance) of Growth Factors in(to) the Nucleus
---
authors: 'Kettler, Gregory C. and Martiny, Adam C. and Huang, Katherine and Zucker, Jeremy and Coleman, Maureen L. and Rodrigue, Sebastien and Chen, Feng and Lapidus, Alla and Ferriera, Steven and Johnson, Justin and Steglich, Claudia and Church, George M. and Richardson, Paul and Chisholm, Sallie W.'
doi: 10.1371/journal.pgen.0030231
title: Patterns and Implications of Gene Gain and Loss in the Evolution of Prochlorococcus
---
authors: 'Kieffer, Martin and Stern, Yaniv and Cook, Holly and Clerici, Elena and Maulbetsch, Christoph and Laux, Thomas and Davies, Brendan'
doi: 10.1105/tpc.105.039107
title: Analysis of the Transcription Factor WUSCHEL and Its Functional Homologue in Antirrhinum Reveals a Potential Mechanism for Their Roles in Meristem Maintenance
---
authors: 'Kiełbasa, Szymon M. and Korbel, Jan O. and Beule, Dieter and Schuchhardt, Johannes and Herzel, Hanspeter'
title: Combining frequency and positional information to predict transcription factor binding sites
---
authors: 'Kim, Hwa-Young and Gladyshev, Vadim N.'
doi: 10.1371/journal.pbio.0030375
title: 'Different Catalytic Mechanisms in Mammalian Selenocysteine- and Cysteine-Containing Methionine-R-Sulfoxide Reductases'
---
authors: 'Kim, Chul-Hyun and Cheng Kao, C. and Tinoco, Jr., Ignacio'
title: RNA motifs that determine specificity between a viral replicase and its promoter
---
authors: 'Kim, Henry S. and Kuwano, Yuki and Zhan, Ming and Pullmann, Jr., Rudolf and Mazan-Mamczarz, Krystyna and Li, Huai and Kedersha, Nancy and Anderson, Paul and Wilce, Matthew C. J. and Gorospe, Myriam and Wilce, Jacqueline A.'
doi: 10.1128/MCB.01036-07
title: Elucidation of a C-Rich Signature Motif in Target mRNAs of RNA-Binding Protein TIAR
---
authors: 'Kim, Jan T. and Martinetz, Thomas and Polani, Daniel'
title: Bioinformatic Principles Underlying the Information Content of Transcription Factor Binding Sites
---
authors: 'Kim, Sung-Kyu and Nam, Jin-Wu and Rhee, Je-Keun and Lee, Wha-Jin and Zhang, Byoung-Tak'
doi: 10.1186/1471-2105-7-411
title: 'miTarget: microRNA target gene prediction using a support vector machine'
---
authors: 'Kim, Jongsook and Shapiro, David J.'
title: In simple synthetic promoters YY1-induced DNA bending is important in transcription activation and repression
---
authors: 'King, Oliver D.'
title: Bounds for DNA codes with constant GC-content
---
authors: 'Kiryu, Hisanori and Kin, Taishin and Asai, Kiyoshi'
title: Robust prediction of consensus secondary structures using averaged base pairing probability matrices
---
authors: 'Kishore, S. and Stamm, S.'
doi: 10.1101/sqb.2006.71.024
title: Regulation of Alternative Splicing by snoRNAs
---
authors: 'Kishore, Shivendra and Stamm, Stefan'
doi: 10.1126/science.1118265
title: The snoRNA HBII-52 Regulates Alternative Splicing of the Serotonin Receptor 2C
---
authors: 'Klamt, Steffen and Saez-Rodriguez, Julio and Gilles, Ernst D'
title: Structural and functional analysis of cellular networks with CellNetAnalyzer
---
authors: 'Klamt, Steffen and Saez-Rodriguez, Julio and Lindquist, Jonathan A and Simeoni, Luca and Gilles, Ernst D'
doi: 10.1186/1471-2105-7-56
title: A methodology for the structural and functional analysis of signaling and regulatory networks
---
authors: 'Klein, P hilip N. and t, Tho mas B. S ebast ia n and t, Benj amin B. Kimia'
title: 'Shape matching using edit-distance: an implementation'
---
authors: 'Klein, Robert J and Eddy, Sean R'
title: 'RSEARCH: Finding homologs of single structured RNA sequences'
---
authors: 'Klein, D. J. and Schmeing, T. M. and Moore, P. B. and Steitz, T. A.'
title: 'The kink-turn: a new RNA secondary structure motif'
---
authors: 'Klemm, Konstantin and Flamm, Christoph and Stadler, Peter F.'
title: Funnels in Energy Landscapes
---
authors: 'Klemm, K. and Flamm, C. and Stadler, P. F.'
doi: 10.1140/epjb/e2008-00136-7
title: Funnels in energy landscapes
---
authors: 'Klingenhoff, Andreas and Frech, Kornelie and Quandt, Kerstin and Werner, Thomas'
title: Functional promoter modules can be detected by formal models independent of overall nucleotide sequence similarity
---
authors: 'Klosterman, Peter S. and Tamura, Makio and Holbrook, Stephen R. and Brenner, Steven E.'
title: 'SCOR: a Structural Classification of RNA database'
---
authors: 'Klug, Stefanie J. and HÜTTENHOFER, Alexander and Famulok, Michael'
title: 'In vitro selection of RNA aptamers that bind special elongation factor SelB, a protein with multiple RNA-binding sites, reveals one major interaction domain at the carboxyl terminus'
---
authors: 'Klug, Stefanie J. and Huttenhofer, Alexander and Kromayer, Matthias and Famulok, Michael'
title: In vitro and in vivo characterization of novel mRNA motifs that bind special elongation factor SelB
---
authors: 'Knight, Rob and Yarus, Michael'
doi: 10.1261/rna.2138803
title: 'Finding specific RNA motifs: Function in a zeptomole world?'
---
authors: 'Knudsen, B. and Hein, J.'
title: RNA secondary structure prediction using stochastic context-free grammars and evolutionary history
---
authors: 'Knudsen, Steen'
title: 'Promoter2.0: for the recognition of PolII promoter sequences'
---
authors: 'Kobayashi, Hideki and Kærn, Mads and Araki, Michihiro and Chung, Kristy and Gardner, Timothy S. and Cantor, Charles R. and Collins, James J.'
doi: 10.1073/pnas.0402940101
title: 'Programmable cells: Interfacing natural and engineered gene networks'
---
authors: 'Koch, Ina'
title: Enumerating all connected maximal common subgraphs in two graphs
---
authors: 'Kochetov, Alex V. and Ponomarenko, Mikhail P. and Frolov, Anatoly S. and Kisselev, Lev L. and Kolchanov, Nikolay A.'
title: Prediction of eukaryotic mRNA translational properties
---
authors: 'Koehl, Patrice and Levitt, Michael'
title: De Novo Protein Design. I. In Search of Stability and Specificity
---
authors: 'Koehl, Patrice and Levitt, Michael'
title: De Novo Protein Design. II. Plasticity in Sequence Space
---
authors: 'Kolb, Fabrice A. and Malmgren, Charlotta and Westhof, Eric and Ehresmann, Chantal and Ehresmann, Bernard and Wagner, E. Gerhart H. and Romby, Pascale'
title: An unusual structure formed by antisense-target RNA binding involves an extended kissing complex with a four-way junction and a side-by-side helical alignment
---
authors: 'Kolb, Fabrice A. and Westhof, Eric and Ehresmann, Bernard and Ehresmann, Chantal and Wagner, E. Gerhart H. and Romby, Pascale'
doi: 10.1006/jmbi.2001.4677
title: 'Four-way Junctions in Antisense RNA-mRNA Complexes Involved in Plasmid Replication Control: A Common Theme?'
---
authors: 'Kolchanov, N. A. and Ponomarenko, M. P. and Frolov, A. S. and Ananko, E. A. and Kolpakov, F. A. and Ignatieva, E. V. and Podkolodnaya, O. A. and Goryachkovskaya, T. N. and Stepanenko, I. L. and Merkulova, T. I. and Babenko, V. V. and Ponomarenko, Yu. V. and Kochetov, A. V. and Podkolodny, N. L. and Vorobiev, D. V. and Lavryushev, S. V. and Grigorovich, D. A. and Kondrakhin, Yu. V. and Milanesi, L. and Wingender, E. and Solovyev, V. and Overton, G. C.'
title: Integrated databases and computer systems for studying eukaryotic gene expression
---
authors: 'Kolinski, Andrzej'
title: Protein modeling and structure prediction with a reduced representation
---
authors: 'Kolinski, Andrzej and Klein, Piotr and Romiszowski, Piotr and Skolnick, Jeffrey'
title: 'Unfolding of Globular Proteins: Monte Carlo Dynamics of a Realistic Reduced Model'
---
authors: 'Kolodny, Rachel and Linial, Nathan'
doi: 10.1073/pnas.0404383101
title: Approximate protein structural alignment in polynomial time
---
authors: 'Konagurthu, Arun Siddharth and Whisstock, James and Stuckey, Peter J.'
title: PROGRESSIVE MULTIPLE ALIGNMENT USING SEQUENCE TRIPLET OPTIMIZATIONS AND THREE-RESIDUE EXCHANGE COSTS
---
authors: 'Kondrakhin, Y. V. and Kel, A. E. and Kolchanov, N. A. and LMilanesi, '
title: Eukaryotic promoter recognition by binding sites for transcription factors
---
authors: 'Kondrashov, Fyodor A. and Koonin, Eugene V.'
title: 'Evolution of alternative splicing: deletions, insertions and origin of functional parts of proteins from intron sequences'
---
authors: 'Kondrashov, Fyodor A. and Koonin, Eugene V.'
title: Origin of alternative splicing by tandem exon duplication
